### DISCUSSION TITLE
Guild Levels and Perks.

### DISCUSSION THREAD

**Discussion by Vyperhand** [chromiecraft/chromiecraft#5676] [Suggestions]
Perhaps we can introduce guild levels and perks as another activity for folks to engage in between content releases, as well as encouraging people to form and actually use guilds and guild runs.

https://wowpedia.fandom.com/wiki/Guild_advancement#Guild_levels_and_perks

This would be a very nice QoL upgrade if it's doable.

**Comment by BeefSteaker** [comment]
Its a Cata Feature, i dont know how doable it would be.
But if someone does manage it would be very cool

**Comment by Aethereus-ManaOnion** [comment]
I think if you found an existing module for AC that does this it could be worth considering, but making it from scratch would take too much effort and time off the developers who are already overburdened

**Comment by Thealiyyu** [comment]
I think it would take much time, but yeahh

**Comment by Kirishiki** [comment]
this is horrible idea, the guild levels and perks was worst thing in cata cos people joined high lvl guilds just for the perks, making new guilds or smaller guilds just die out or generally VERY hard to get people to join. this whole thing would have too big of impact of current guilds and break guild bonds that weve created trough chromie era.

**Reply by shaszun** [reply]
Never played cata, but this sound like the most logical outcome of such a change. Not to mentione the time required to implement, which will make Tiers releases and bug fixes slower.
